src/components/Dropdown.tsx

```diff
@@ -43,9 +86,13 @@ const options = {
     properties: DropdownPropsSchema,
     state: DropdownStateSchema,
     methods: {},
-    slots: ["trigger", "list"],
+    slots: ['trigger'],
     styleSlots: [],
-    events: [],
+    events: [
+      'onClickMenuItem', 
+      'onVisibleChange',
+      'onButtonClick'
```

Comment:
which button?

Reply:
When set `type` property to `button`, it will use `Dropdown.Button` component. A button will locate at left side.

![image](https://user-images.githubusercontent.com/25414733/150244834-e990e81c-d879-494b-b3b1-3b8c1d8712d6.png)

Reply:
ok